bindings/ergo-lib-wasm/src/block_header.rs

```diff
@@ -33,10 +33,26 @@ impl BlockHeader {
 #[derive(PartialEq, Eq, Debug, Clone, From, Into)]
 pub struct BlockId(pub(crate) ergo_lib::ergo_chain_types::BlockId);
 
+#[wasm_bindgen]
+impl BlockId {
+    /// Parse from base 16 encoded string
+    #[allow(clippy::should_implement_trait)]
+    pub fn from_str(id: &str) -> Result<BlockId, JsValue> {
+        ergo_lib::ergo_chain_types::Digest32::try_from(String::from(id))
+            .map(|d| BlockId(ergo_lib::ergo_chain_types::BlockId(d)))
+            .map_err(to_js)
+    }
+
+    /// Equality check
+    pub fn equals(&self, id: &BlockId) -> bool {
```

Comment:
This made me realize that we might need to implement a proper equality check like this one for all other types. 
The types that have JSON encoding are in less of a hurry though.